igvm/hypervisor.py

```diff
@@ -536,6 +536,7 @@ def migrate_vm(
                         'DRBD migration is supported only between hypervisors '
                         ' using LVM storage!'
                     )
+                vm.safe_blocksize()
```

Comment:
I think we should find a different name for it. It currently could look like a getter (get me the safe blocksize for this vm). We already have some more methods starting with set_, so maybe use set_safe_blocksize()

Reply:
Sure, I can fix that.